### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jaggs6**
Fix typo

---
This change is [Reviewable](https://reviewable.io/reviews/chrisledet/rebasebot/10)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ A GitHub bot that rebases your pull request branches when you ask
 1. Make a dedicated GitHub account for the bot
 2. Grant the GitHub account read and write access to your repositories
 3. [Setup](#setup) the bot on your own server
-4. **@{github bot username} rebase** the bot in a pull request comment
+4. Type a comment **@{github bot username} rebase** in a pull request
```
Comment (chrisledet) [inline README.md]: How about wrapping it with quotes as well?

Example:
Type a comment "**@{github bot username} rebase**" in a pull request

**Comment by jaggs6** [comment]
@chrisledet sorry it took so long. done the change

**Comment by jaggs6** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[README.md, line 10 at r1](https://reviewable.io:443/reviews/chrisledet/rebasebot/10#-KbLmn0-XPQI-dxn8PIh-r1-10:-KbLnhJiurV0tPzwKrD1:b-896fix) ([raw file](https://github.com/chrisledet/rebasebot/blob/f9caf0d608300090d0c74be1da693ef38ce2d23c/README.md#L10)):*
<details><summary><i>Previously, chrisledet (Chris Ledet) wrote…</i></summary><blockquote>

How about wrapping it with quotes as well?

Example:
Type a comment "**@{github bot username} rebase**" in a pull request

</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/chrisledet/rebasebot/10)*

**Review by chrisledet — APPROVED** [review]

**Comment by chrisledet** [comment]
Thanks!

**Pull request merged**